operator/data/profiles/default.yaml

```diff
@@ -468,17 +510,7 @@ spec:
       istio-egressgateway:
         autoscaleEnabled: true
         type: ClusterIP
-        env:
-          ISTIO_META_ROUTER_MODE: "sni-dnat"
```

Comment:
I am not 100% positive, but I believe this will effect multicluster egress... @ayj would you mind taking a look?

Reply:
this is moved to iop k8s settings, not removed.

Reply:
Ok, I can live with that answer. I don't see a corresponding value for default.yaml _by default_. Can you point me at them, or are we expecting our users to know how to set environment variables that are undocumented?

Reply:
see my pointer in a separate comment. thanks for reviewing carefully though, this definitely needs careful inspection.

Reply:
LGTM.